extensions/blocks/contact-form/save.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export default function Save( { attributes } ) {
+	const { postUrl } = attributes;
+	return <a href={ postUrl }>{ __( 'Submit a contact form.', 'jetpack' ) }</a>;
```

Comment:
Introducing translatable strings in saved content may cause some issues when switching languages on a site. You can learn more about this here: #16361 

Do you think we could avoid this approach here?

Reply:
@kwight We can do the same we did for the "Pay with PayPal" block here and get the text from a block attribute:

https://github.com/Automattic/jetpack/blob/b7ccbb1d60d8d0a1463711836a810d1d5f012728/extensions/blocks/simple-payments/index.js#L123-L128

https://github.com/Automattic/jetpack/blob/b7ccbb1d60d8d0a1463711836a810d1d5f012728/extensions/blocks/simple-payments/save.js#L55

The text can be editable in the block sidebar:

https://github.com/Automattic/jetpack/blob/b7ccbb1d60d8d0a1463711836a810d1d5f012728/extensions/blocks/simple-payments/edit.js#L425-L441

Reply:
@kwight I incorporated @mmtr 's suggestions but haven't tested them. Specifically, I'm unsure of the changes I made to `extensions/blocks/contact-form/edit.js`